source/_includes/3.3/documentation-preferences-profiles-terminal.md

```diff
@@ -1,4 +1,4 @@
-## Profiles
+Profiles
```

Comment:
`##` makes it format as a large title. Without it I think it would be small, like regular paragraph text.

Reply:
good catch. This was a mistake from editing.